cli/wallet/multisig.go

```diff
@@ -84,10 +88,15 @@ func signStoredTransaction(ctx *cli.Context) error {
 		if err != nil {
 			return cli.NewExitError(fmt.Errorf("failed to submit transaction to RPC node: %w", err), 1)
 		}
-		fmt.Fprintln(ctx.App.Writer, res.StringLE())
-		return nil
+		if ctx.Bool("await") {
+			version, err := c.GetVersion()
+			aer, err = waiter.New(c, version).Wait(res, tx.ValidUntilBlock, err)
+			if err != nil {
+				return cli.NewExitError(fmt.Errorf("the transaction doesn't accepted to the block: %w", err), 1)
```

Comment:
Please, fix this error everywhere. It doesn't mean that transaction wasn't accepted. It means in general that transaction awaiting failed. And transaction itself is sent to the network, it may be accepted or may not be accepted.